src/System.Net.WebHeaderCollection/ref/System.Net.WebHeaderCollection.cs

```diff
@@ -90,32 +90,40 @@ public partial class WebHeaderCollection : System.Collections.Specialized.NameVa
         protected WebHeaderCollection(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
         public override string[] AllKeys { get { throw null; } }
         public override int Count { get { throw null; } }
-        public string this[System.Net.HttpRequestHeader header] { get { throw null; } set { } }
-        public string this[System.Net.HttpResponseHeader header] { get { throw null; } set { } }
-        public new string this[string name] { get { throw null; } set { } }
+        public string? this[System.Net.HttpRequestHeader header] { get { throw null; } set { } }
+        public string? this[System.Net.HttpResponseHeader header] { get { throw null; } set { } }
+        public new string? this[string name] { get { throw null; } set { } }
         public override System.Collections.Specialized.NameObjectCollectionBase.KeysCollection Keys { get { throw null; } }
-        public void Add(System.Net.HttpRequestHeader header, string value) { }
-        public void Add(System.Net.HttpResponseHeader header, string value) { }
+        public void Add(System.Net.HttpRequestHeader header, string? value) { }
+        public void Add(System.Net.HttpResponseHeader header, string? value) { }
         public void Add(string header) { }
-        public override void Add(string name, string value) { }
-        protected void AddWithoutValidate(string headerName, string headerValue) { }
+#pragma warning disable CS8610
```

Comment:
Should we disable it for the whole file? 

If people ever need to re-generate the ref assembly source, having all this instances, will cause the diff to be harder to get right and will take longer for people to fix it.

Reply:
I am fine with disabling the whole file, what if in between methods/properties updated/added could cause/have a valid warning?